compiler/executable/pipeline.rs

```diff
@@ -534,3 +552,113 @@ fn find_referenced_functions_in_fetch(
         })
     }
 }
+
+#[derive(Debug, Clone)]
+pub struct ParametrisedQueryStructure {
+    pub branches: [Option<Vec<Constraint<Variable>>>; 64],
```

Comment:
are we limited to 64 branches throughout the entire query now? Can we relax this?